python/eos/tasks.py

```diff
@@ -460,6 +460,64 @@ def sample_nested(analysis_file:str, posterior:str, base_directory:str='./', bou
     eos.data.DynestyResults.create(os.path.join(base_directory, posterior, 'dynesty_results'), analysis.varied_parameters, results)
 
 
+# Corner plot
+@task('corner-plot', '{posterior}/plots')
+def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
+    """
+    Generates a corner plot of the 1-D and 2-D marginalized posteriors.
+
+    The input files are expected in EOS_BASE_DIRECTORY/POSTERIOR/samples.
+    The output files will be stored in EOS_BASE_DIRECTORY/POSTERIOR/plots.
+
+    :param analysis_file: The name of the analysis file that describes the named posterior, or an object of class `eos.AnalysisFile`.
+    :type analysis_file: str or `eos.AnalysisFile`
+    :param posterior: The name of the posterior.
+    :type posterior: str
+    :param base_directory: The base directory for the storage of data files. Can also be set via the EOS_BASE_DIRECTORY environment variable.
+    :type base_directory: str, optional
+    """
+    p = eos.Parameters()
```

Comment:
I think we should

1. Create the analysis,
2. Use its parameters object rather than the defaults.

Would make the task usable even when we let an analysis modify the latex representations of its parameters.